libs/community/langchain_community/chat_models/yuan2.py

```diff
@@ -0,0 +1,506 @@
+"""ChatYuan2 wrapper."""
+from __future__ import annotations
+
+import asyncio
+import logging
+import os
+
+os.environ["no_proxy"] = "localhost,127.0.0.1,::1"
```

Comment:
delete

Reply:
Sure, I'll fix this, Thanks.

Reply:
@baskaryan I've deleted unnecessary import and related code block, it still stuck in deploying, did I miss anything? Thanks, PTAL :)